pkg/controller/job/metrics/metrics.go

```diff
@@ -84,6 +86,18 @@ var (
 		},
```

Comment:
update the help text above to indicate that pods that fit the "ignore" rule are not counted.